lib/src/main/java/co/nimblehq/alpine/lib/AlpineLib.kt

```diff
@@ -0,0 +1,55 @@
+package co.nimblehq.alpine.lib
+
+import android.content.Context
+import android.nfc.Tag
+import co.nimblehq.alpine.lib.model.CameraImage
+import co.nimblehq.alpine.lib.model.MrzInfo
+import co.nimblehq.alpine.lib.mrz.MrzProcessor
+import co.nimblehq.alpine.lib.mrz.MrzProcessorImpl
+import co.nimblehq.alpine.lib.mrz.MrzProcessorResultListener
+import co.nimblehq.alpine.lib.nfc.ISO_DEP_TIMEOUT_IN_MILLIS
+import co.nimblehq.alpine.lib.nfc.NfcReader
+import co.nimblehq.alpine.lib.nfc.NfcReaderImpl
+import com.google.mlkit.vision.common.InputImage
+
+object AlpineLib {
+
+    private val nfcReader: NfcReader = NfcReaderImpl()
+    private val mrzProcessor: MrzProcessor = MrzProcessorImpl()
+
+    fun initializeNfcReader(context: Context) = nfcReader.initialize(context)
+
+    fun readNfc(
+        tag: Tag,
+        mrzInfo: MrzInfo,
+        timeout: Int = ISO_DEP_TIMEOUT_IN_MILLIS
+    ) = nfcReader.readNfc(
```

Comment:
So this would become 
```suggestion
    fun readNfc(
        context: Context,
        tag: Tag,
        mrzInfo: MrzInfo,
        timeout: Int = ISO_DEP_TIMEOUT_IN_MILLIS
    ) = nfcReader.readNfc(
```
and the other methods would follow 🤔